### ISSUE TITLE
[Usable] Name of controls are not properly defined after activate the 'Help' command.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1408]
**Actual Result:**
Name of controls are not properly defined after activate the 'Help' command. reading 'twice' all control names.

**Expected Result:**
Name of controls should be properly defined after activate the 'Help' command. 

**Repro Steps:**

1. Open URL [https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmicrosoft.github.io%2FBotFramework-WebChat%2Fspeech-cognitive-services-bing-speech%2F%3Fs%3D7c7dacfe04044f4f99a1edcf9b0e2ab9&data=02%7C01%7Cv-saugup%40microsoft.com%7C0f6f5e63167449daa46d08d65570f676%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636790341026155420&sdata=gcFq8JsflCLfUKAyw8f2A19aJcEQEFxGzjASMK9S59A%3D&reserved=0](https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmicrosoft.github.io%2FBotFramework-WebChat%2Fspeech-cognitive-services-bing-speech%2F%3Fs%3D7c7dacfe04044f4f99a1edcf9b0e2ab9&data=02%7C01%7Cv-saugup%40microsoft.com%7C0f6f5e63167449daa46d08d65570f676%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636790341026155420&sdata=gcFq8JsflCLfUKAyw8f2A19aJcEQEFxGzjASMK9S59A%3D&reserved=0) in Edge browser.
2.  Notice 'Upload File' and 'Mic' button icons present in bottom of screen.
3. Navigate to mic button by using Tab key and press Enter to activate it.
4. Mic button will activate and said 'Help'.
5. New content will appear on the screen.
6. Observe that after activate the help command new content will appear on the screen and reading all group names twice.
    
**User Impact:**
Screen reader users may get confuse if same content is being read more than once.

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge

**Attachment:**
[Usable_reading 2 times.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2628393/Usable_reading.2.times.zip)

**Comment by corinagum** [comment]
Can be fixed at same time as https://github.com/Microsoft/BotFramework-WebChat/issues/1409

**Comment by corinagum** [comment]
@priysn @compulim I am not able to repro at this time. Could you try it out?

**Comment by compulim** [comment]
Saw 2 MP3s going on in the network trace.

**Comment by compulim** [comment]
Looks like we fixed in recent PRs around speech. I don't see 2 x MP3s going on now.

Please feel free to comment and we will reopen this one.

![image](https://user-images.githubusercontent.com/1622400/50364477-ef1b0e00-0524-11e9-97d2-b7ba7dbe928f.png)

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yUsable

**Comment by pgopinadhu** [comment]
#A11yUsable;#A11ySev4;
